core/primitives/src/runtime/parameter_table.rs

```diff
@@ -60,10 +60,54 @@ impl ParameterValue {
     }
 }
 
+fn format_number(mut n: u64) -> String {
+    let mut parts = Vec::new();
+    while n >= 1000 {
+        parts.push(format!("{:03?}", n % 1000));
+        n /= 1000;
+    }
+    parts.push(n.to_string());
+    parts.reverse();
+    parts.join("_")
+}
+
+impl core::fmt::Display for ParameterValue {
+    fn fmt(&self, f: &mut core::fmt::Formatter) -> core::fmt::Result {
+        match self {
+            ParameterValue::U64(v) => write!(f, "{}", format_number(*v)),
+            ParameterValue::Rational { numerator, denominator } => {
+                write!(f, "{numerator} / {denominator}")
+            }
+            ParameterValue::Fee { send_sir, send_not_sir, execution } => {
+                write!(
+                    f,
+                    r#"
+- send_sir:     {}
+- send_not_sir: {}
+- execution:    {}"#,
```

Comment:
if you decide to right align the other numbers, probably also here

Reply:
Done.